.golangci.yaml

```diff
@@ -0,0 +1,135 @@
+run:
+  timeout: 4m
+
+linters:
+  disable-all: false
+  enable:
+    - gocyclo
+    - misspell
+    - nilerr
+    - unparam
+    - gosec
+    - unused
+    - govet
+    - gosimple
+    - errorlint
+    - errcheck
+    - dogsled
+    - cyclop
+    - exhaustive
+    - nestif
+    - goconst
+    - gofmt
```

Comment:
I think it's worth switching to `gofumpt` here as it contains more formatting rules (backwards compatible with `gofmt`) and import sorting. I've been using it for quite a while, very happy with the results.